tox.ini

```diff
@@ -27,36 +28,36 @@ setenv =
     PYTHONPATH = {toxinidir}/tests:{toxinidir}
     PYTHONDEVMODE = 1
 
-deps =
-    flit>=3.9
+extras = testing
 
-    django3.2: Django>=3.2,<3.3
+deps =
     django4.2: Django>=4.2,<4.3
     django5.0: Django>=5.0,<5.1
-    djmain: git+https://github.com/django/django.git@main#egg=Django
 
-    wagtail4.1: wagtail>=4.1,<4.2
     wagtail5.2: wagtail>=5.2,<5.3
+    wagtail6.0: wagtail>=6.0,<6.1
     wagtailmain: git+https://github.com/wagtail/wagtail.git
 
     postgres: psycopg2>=2.9
 
-    .[testing]
-
```

Comment:
@zerolab Are there not any other test deps that we'd want to install here?

Reply:
This moved to the `extras` line, which tox then automatically includes in the frozen deps calculation

Reply:
Ah, I see it now. Thanks for confirming!